src/pyj/read_book/flow_mode.pyj

```diff
@@ -57,7 +57,7 @@ last_change_spine_item_request = {}
 
 def _check_for_scroll_end(func, obj, args, report):
     before = window.pageYOffset
-    func.apply(obj, args)
+    should_flip_progression_direction = func.apply(obj, args)
```

Comment:
Different movement functions may or may not want to reverse the page progression direction. For example, the scrollwheel down is naturally "next page" even in RTL books, but clicking on the left margin should go forward in RTL mode and backward in LTR mode.